cranelift/filetests/src/test_zkasm.rs

```diff
@@ -73,6 +73,8 @@ mod tests {
             }
         }
 
+        // The total amount of stack available on ZKASM processor is 2^16 of 8-byte words.
```

Comment:
If its a separate memory address space, should definitely mention it at every possible opportunity. If its the same one, the amount available is shared with the heap which (likely) grows upward from address 0.

Reply:
It is a separate memory space that is not shared with the heap. I will edit the comment to mention it.